spec/configuration/prison_data_spec.rb

```diff
@@ -20,18 +20,48 @@
   context "enabled prisons" do
     let(:enabled) { Prison.enabled }
 
+    it "currently number between 95 and 105" do
+      expect(enabled.count).to be_within(5).of(100)
+    end
```

Comment:
I don't really like this approach, but I need to be sure that somebody doesn't accidentally unplug the loading of the live prison data.  If that happened, then some of the specs may well start to throw false positives.